tests/fixtures/unit/configuration_subsystem/ansible-navigator.yml

```diff
@@ -2,9 +2,12 @@
 ansible-navigator:
   ansible:
     config: /tmp/ansible.cfg
+    cmdline: "--forks 15"
```

Comment:
Oh, I just spotted something. This isn't a change specific to this PR since you just moved lines, but I noticed the value of "{{ ansible_forks }}" isn't correct unless this is actually a list: `cmdline: ["--forks", "15"]`

Reply:
I'll add an issue to check this, I'm 99.6% sure we shlex split them on the way to runner